src/Integration.Vsix.UnitTests/CFamily/CFamilyHelperTests.cs

```diff
@@ -120,6 +120,7 @@ public void CreateRequest_NoAnalyzerOptions_RequestCreatedWithoutOptions()
             (request.Flags & Request.BuildPreamble).Should().Be(0);
             request.RulesConfiguration.Should().NotBeNull();
             request.Options.Should().NotBeEmpty();
+            request.AnalyzerOptions.Should().BeNull();
```

Comment:
This file contains for "create request". I've fixed them up to check that `IRequest.AnalyzerOptions` is set appropriately.